<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite thread-count="2" guice-stage="DEVELOPMENT" name="Failed suite [Suite]" parallel="methods">
  <test thread-count="2" name="Test(failed)" parallel="methods">
    <classes>
      <class name="qaautomation.march2022.AppTestAfterPOM">
        <methods>
          <include name="loginInvalidPassword"/>
          <include name="setUp"/>
          <include name="cleanUp"/>
        </methods>
      </class> <!-- qaautomation.march2022.AppTestAfterPOM -->
      <class name="qaautomation.march2022.AppTestAfterPageFactory">
        <methods>
          <include name="cleanUp"/>
          <include name="setUp"/>
          <include name="loginInvalidPassword"/>
        </methods>
      </class> <!-- qaautomation.march2022.AppTestAfterPageFactory -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
